src/templates/homepage/AnnouncementsSection.tsx

```diff
@@ -24,7 +25,10 @@ export const TemplateAnnouncementsSection = forwardRef<
       subtitle,
       announcementItems,
       sectionIndex,
-    }: TemplateAnnouncementsSectionProps,
+      announcementScrollRefs,
+    }: TemplateAnnouncementsSectionProps & {
+      announcementScrollRefs: Ref<HTMLDivElement>[]
+    },
```

Comment:
supernit: this is extra alr since `TemplateAnnouncementsSectionProps` has the property.

Reply:
ps forgot to rm 54192b920cf2caf20804575260bfe5bfbe1178b1